docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -592,6 +592,24 @@ public class CustomTokenStateManager implements TokenStateManager {
 }
 ----
 
+=== Proof Of Key for Code Exchange (PKCE)
+
+link:https://datatracker.ietf.org/doc/html/rfc7636[Proof Of Key for Code Exchange] (PKCE) minimizes the risk of the authorization code interception.
+
+While `PKCE` is of primary importance to the public OpenId Connect clients (such as the SPA scripts running in a browser), it can also provide an extra level of protection to Quarkus OIDC `web-app` applications which are confidential OpenId Connect clients capable of securely storing the client secret and using it to exchange the code for the tokens.
```

Comment:
@pedroigor No, we don't send `nonce` yet, I'll open an issue to track it, it will be easier to handle it alongside PKCE code verifier as both could be encrypted alongside each other in a state cookie

Reply:
I see. PKCE, should be enough I think? The only thing that makes me think `nonce` is important is when the OP does not support PKCE.

Reply:
@pedroigor Sure, it will be optional for `web-app` I guess, or required with a strict profile